storybookjs/storybook
# CLI: Update renderer templates to provide correct typescript examples

Closes #21179



## What I did

There was a change in the naming of directories inside of the CLI where `template/cli/ts` was split into `template/cli/ts-3-8` and `template/cli-ts-4-9`, but not all renderers had the directories renamed. This does that for:
- html
- web-components
- svelte

There's also Preact, which doesn't have typescript examples at all, but given it involves a little extra work (there's some conflicts with React errors), I'd say it could be done in a separate PR.

## How to test

1. Run a sandbox for template, e.g. `yarn task --task sandbox --start-from auto --template html-vite/default-ts`
2. Check that the generated stories are typescript

## Checklist



      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers

      make sure to add the `ci:merged` or `ci:daily` GH label to it.

`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
